### ISSUE TITLE
MySQL Example not working

### ISSUE THREAD

**Issue by fabianbaier** [kudobuilder/frameworks#25]
When trying to follow https://github.com/maestrosdk/maestro/blob/master/docs/Backups.md and run

```
maestro-demo $ kubectl apply -f mysql.yaml
framework.maestro.k8s.io/mysql created
frameworkversion.maestro.k8s.io/mysql-57 created
instance.maestro.k8s.io/mysql created
```

I get the following error message:
```
maestro-demo $ kubectl logs mysql-mysql-77fcd5d94f-bvqmx
Initializing database
2019-01-29T05:54:00.352589Z 0 [Warning] TIMESTAMP with implicit DEFAULT value is deprecated. Please use --explicit_defaults_for_timestamp server option (see documentation for more details).
2019-01-29T05:54:00.353999Z 0 [ERROR] --initialize specified but the data directory has files in it. Aborting.
2019-01-29T05:54:00.354032Z 0 [ERROR] Aborting
```

Log:
```
mysql-mysql-77fcd5d94f-bvqmx   0/1   Pending   0     0s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Pending   0     0s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Pending   0     1s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Pending   0     3s
mysql-mysql-77fcd5d94f-bvqmx   0/1   ContainerCreating   0     3s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Error   0     14s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Error   1     15s
mysql-mysql-77fcd5d94f-bvqmx   0/1   CrashLoopBackOff   1     16s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Error   2     30s
mysql-mysql-77fcd5d94f-bvqmx   0/1   CrashLoopBackOff   2     38s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Error   3     54s
mysql-mysql-77fcd5d94f-bvqmx   0/1   CrashLoopBackOff   3     58s
mysql-mysql-77fcd5d94f-bvqmx   0/1   Error   4     104s
mysql-mysql-77fcd5d94f-bvqmx   0/1   CrashLoopBackOff   4     105s
```

**Comment by yankcrime** [comment]
Looks like I missed this issue when I proposed #65, which fixes the problem for the MySQL operator albeit in a different way to the one outlined above.

**Comment by guenter** [comment]
Fixed by #169
